extension-manifest-v3/src/background/onboarding.js

```diff
@@ -26,9 +31,28 @@ observe('onboarding', (onboarding) => {
 });
 
 if (__PLATFORM__ === 'chromium' && isOpera()) {
-  chrome.webNavigation.onCompleted.addListener((details) => {
+  const NOTIFICATION_DELAY = 7 * 24 * 60 * 60 * 1000; // 7 days in milliseconds
+  const NOTIFICATION_SHOW_LIMIT = 4;
+
+  chrome.webNavigation.onCompleted.addListener(async (details) => {
```

Comment:
can we please move this code outside of the onboarding?

Reply:
Ok, I'll create an opera.js file for that.